### ISSUE TITLE
Drop View And Create again throws error 'cannot drop view employeeview because other objects depend on it' 

### ISSUE THREAD

**Issue by psrinivasa2** [wiltondb/wiltondb#96]
Hi @staticlibs ,
We have encountered a issue today that Drop View And Create again throws error 'cannot drop view employeeview because other objects depend on it'. We have referenced the view in eight different other views. 

It is important to note that the same error not occurs in SQL side. 

As Alter View not supported in babelfish, we will normally drop the view and recreate it, but in this case we need to drop all the depended view and then drop base view which is not providing flexibility. 

Can you please look at it?

**Comment by staticlibs** [comment]
@psrinivasa2 

Yes, `ALTER VIEW` is not supported in latest Babelfish yet. Though there is some activity on this - [#3076](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/3076).
